projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-saved-cart.ts

```diff
@@ -301,6 +301,87 @@ export function verifyCartDetails(cart: any) {
     });
 }
 
+export function getCartItem(name: string) {
+  return cy.get('cx-cart-item-list').contains('.cx-item-list-row', name);
+}
+
+export function restoreSavedCart(cart: any) {
+  const restoreSavedCartAlias = interceptRestoreSavedCartEndpoint(cart.code);
+  const getAllSavedCartAlias = interceptGetAllSavedCartEndpoint();
+
+  cy.get('cx-saved-cart-details-action .btn-primary').click();
+
+  cy.get('cx-saved-cart-form-dialog').within(() => {
+    cy.get('button[aria-label="Restore"]').click();
+  });
+
+  cy.wait(`@${restoreSavedCartAlias}`)
+    .its('response.statusCode')
+    .should('eq', 200);
```

Comment:
```suggestion
  function waitForSuccessfulServerResponse(alias: string) {
     cy.wait(alias)
        .its('response.statusCode')
        .should('eq', 200);
  }
  waitForSuccessulServerResponse(restoreSavedCartAlias)
```

If possible modify interceptRestoreSavedCartEndpoint to return `@${restoreSavedCartAlias}`